application/layouts/scripts/body.phtml

```diff
@@ -96,3 +104,7 @@ if ($this->layout()->inlineLayout) {
   ?></ul>
   <div id="application-state-summary" class="container" data-icinga-url="<?= $this->url('application-state/summary') ?>" data-last-update="-1" data-icinga-refresh="60"></div>
 </div>
+<div id="mobile-menu" data-last-update="-1" data-base-target="_main" class="container"
+     data-icinga-url="<?= $this->href('layout/mobile-menu') ?>"  data-icinga-refresh="15">
+    <?= new MobileMenu() ?>
+</div>
```

Comment:
Is it possible to have this reside inside `#footer`? I'd prefer that, knowing that there can't be anything else below it.

Reply:
✅